source/contracts/reporting/IMarket.sol

```diff
@@ -10,7 +10,8 @@ import 'reporting/IDisputeWindow.sol';
 import 'trading/IShareToken.sol';
 import 'reporting/IReportingParticipant.sol';
 import 'reporting/IReputationToken.sol';
-//import 'reporting/IInitialReporter.sol';
+import 'IController.sol';
+import 'IControlled.sol';
```

Comment:
This file added these two imports, but there are no other changes to the file, suggesting that these imports are not necessary?